### ISSUE TITLE
Annoying proptype warning

### ISSUE THREAD

**Issue by sebastianoblom** [clauderic/react-sortable-hoc#492]
Getting this warning;
```
Warning: Invalid argument supplied to oneOfType, expected an instance of array.
```

Seems to be related to the new helperContainer thing I just read about in another issue (stacktrace goes here)
```
helperContainer: PropTypes.oneOfType(PropTypes.func, typeof HTMLElement === 'undefined' ? PropTypes.any : PropTypes.instanceOf(HTMLElement))
```

Version;
```
"react-sortable-hoc": "^1.5.1",
```

**Comment by clauderic** [comment]
Fixed in react-sortable-hoc@1.5.2, sorry about that!

**Comment by sebastianoblom** [comment]
Np, thanks for the quick fix! :)
